src/deprecated.jl

```diff
@@ -2,3 +2,14 @@
 
 @deprecate confint(obj::LinearModel, level::Real) confint(obj, level=level)
 @deprecate confint(obj::AbstractGLM, level::Real) confint(obj, level=level)
+
+function Base.getproperty(mm::LinPredModel, f::Symbol)
+    if f === :model
+        Base.depwarn("accessing the `model` field of GLM.jl models is deprecated, " *
+                     "as they are no longer wrapped in a `TableRegressionModel` " *
+                     "and can be used directly now", :getproperty)
+        return mm
+    else
+        return getfield(mm, f)
+    end
+end
```

Comment:
missing final newline.

Are there other TableRegressionModel fields we should warn about?

Reply:
I've added a branch to allow accessing the formula. Other fields are not very likely to be used, except maybe the data but we probably don't want to store it.